### DISCUSSION TITLE
pfe-menu

### DISCUSSION THREAD

**Discussion by phillip-kruger** [patternfly/patternfly-elements#2113] [Q&A]
Hi all
I am looking to rebuild the [Quarkus Dev UI](https://quarkus.io/guides/dev-ui) with patternflyelements, but I need a menu similar to https://www.patternfly.org/v4/components/menu/

Is that component in the pipeline to create as a web component ?

Thanks

**Comment by bennypowers** [comment]
Its not currently prioritized on our release milestone, but we're interested in doing it.

@markcaron can we add this to the release milestone, considering that we have interest here?
